browser/scripts/graph.js

```diff
@@ -28,14 +28,19 @@ Graph.prototype.update = function() {
 	
 	for(i = 0, len = nodes.length; i < len; i++)
 		nodes[i].update_count = 0
+
+	var updateContext = {
+		abs_t: E2.app.player.core.abs_t,
```

Comment:
if you want to recurse with a specific updateContext, maybe it's better to pass it into Graph.update() as well, since that can also recurse (in nested patches)